examples/benchmarks/bubblebench/experimental-tree/src/proxy/tree.ts

```diff
@@ -197,8 +199,7 @@ export class TreeArrayProxy<T> {
 
 	map<U>(
 		callbackfn: (value: Serializable<T>, index: number, array: Serializable<T>[]) => U,
-		thisArg?: any,
 	): U[] {
-		return this.items.map<U>(callbackfn, thisArg);
+		return this.items.map<U>((value, index, array) => callbackfn(value, index, array));
```

Comment:
I can't tell just by looking at it if this is correct but I'll trust the linter.

Reply:
Whatever it was that made this start failing, that's what I was unconsciously referring to 😝 .

Reply:
This actually wasn't the point of failure. I reverted it briefly to verify. The actual cause was `isNaN` -> `Number.isNaN` given string input.